docs/UserGuide.md

```diff
@@ -126,7 +126,7 @@ Here are some general rules to follow when entering prefixes and parameters:
   
 * To add multiple parameters of the same prefix, add the prefix multiple times before each parameter.<br>
   e.g. To add two TAGs, enter `-t tagOne -t tagTwo`.<br>
-  e.g. To add three HEADERSs, enter `-h \"header:one\" -h \"header:two\" -h \"header:three\"`.<br>
+  e.g. To add three HEADERSs, enter `-h "header:one" -h "header:two" -h "header:three"`.<br>
```

Comment:
Might be better to include whitespaces between `:` and `one`, `two` and `three` to be consistent with the header formatting.